src/components/searchForm/searchForm.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../styles/variables';
+
+.search-container {
+  position: relative;
+}
+
+.search-banner {
+  width: 100%;
+  height: 240px;
+  background: url("../../../public/assets/img/banner.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  position: relative;
+  top: 0px;
+  left: 0px;
+  overflow: hidden;
+  filter: blur(8px);
+  -webkit-filter: blur(8px);
+  opacity: .3;
+}
+
+.search-form {
+  display: flex;
+  align-items: center;
+  margin: 10px;
+  width: 80%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  translate: -50% -50%;
+}
+
+/* Style for the search input */
+.search-box {
+  padding: 10px;
+  border: 1px solid #424242;
+  border-radius: 4px;
+  font-size: 1rem;
+  flex-grow: 1;
+  margin-right: 10px;
+  background-color: $background-color;
+  color: white;
+  opacity: .7;
+}
+
+/* Style for the search button */
+.btn-submit {
+  padding: 10px 20px;
+  background-color: #f65261;
+  /* Change to your preferred button color */
+  color: #fff;
+  /* Text color */
+  border: none;
+  border-radius: 4px;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+/* Hover effect for the search button */
+.btn-submit :hover {
+  background-color: #0056b3;
+  /* Change to your preferred hover color */
+}
```

Comment:
Nest Parent -> Child

Reply:
Nested style according to html structure.